### DISCUSSION TITLE
The calculated volume fractions of S component for fcc are different when using 1 and 222 specimenSymmetry

### DISCUSSION THREAD

**Discussion by Brian-Liu-rx** [mtex-toolbox/mtex#1765] [Ask Anything]
Hi, everyone,
     The specimenSymmetry in Mtex is really hard for me to deal with.
     why the calculated volume fractions of S component for fcc are different when using 1 and 222 specimenSymmetry？
     I need your help. :)

CASE 1:
    CS = crystalSymmetry('m-3m', [4 4 4], 'mineral', 'Aluminum');
    SS = specimenSymmetry('222');       

    psi = deLaValleePoussinKernel('halfwidth',5*degree);
    odf = ODF.load('pf_data.txt',CS,SS,'density','kernel',psi,'resolution',1*degree,...
      'interface','generic',...
      'ColumnNames', { 'phi1' 'Phi' 'phi2' 'Weight'}, 'Columns',[1 2 3 4], 'Bunge');

    ori_s = orientation.byMiller([1 2 3],[6 3 4],CS,SS);
    volume_s =  volume(odf,ori_s,10*degree)       % Here the ans is 0.3

CASE 2:
    CS = crystalSymmetry('m-3m', [4 4 4], 'mineral', 'Aluminum');
    SS = specimenSymmetry('1');      %changed

    psi = deLaValleePoussinKernel('halfwidth',5*degree);
    odf = ODF.load('pf_data.txt',CS,SS,'density','kernel',psi,'resolution',1*degree,...
      'interface','generic',...
      'ColumnNames', { 'phi1' 'Phi' 'phi2' 'Weight'}, 'Columns',[1 2 3 4], 'Bunge');

    ori_s = orientation.byMiller([1 2 3],[6 3 4],CS,SS);
    volume_s =  volume(odf,ori_s,10*degree)       % Here the ans is 0.07

**Comment by lfgmorales** [comment]
Maybe because one has a triclinic sample symmetry and the other orthorhombic?



Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 13 Mar 2023, at 14:06, Brian-Liu-rx ***@***.***> wrote:

﻿

Hi, everyone,
The specimenSymmetry in Mtex is really hard for me to deal with.
why the calculated volume fractions of S component for fcc are different when using 1 and 222 specimenSymmetry？
I need your help. :)

CASE 1:
CS = crystalSymmetry('m-3m', [4 4 4], 'mineral', 'Aluminum');
SS = specimenSymmetry('222');

psi = deLaValleePoussinKernel('halfwidth',5*degree);
odf = ODF.load('pf_data.txt',CS,SS,'density','kernel',psi,'resolution',1*degree,...
  'interface','generic',...
  'ColumnNames', { 'phi1' 'Phi' 'phi2' 'Weight'}, 'Columns',[1 2 3 4], 'Bunge');

ori_s = orientation.byMiller([1 2 3],[6 3 4],CS,SS);
volume_s =  volume(odf,ori_s,10*degree)       % Here the ans is 0.3


CASE 2:
CS = crystalSymmetry('m-3m', [4 4 4], 'mineral', 'Aluminum');
SS = specimenSymmetry('1'); %changed

psi = deLaValleePoussinKernel('halfwidth',5*degree);
odf = ODF.load('pf_data.txt',CS,SS,'density','kernel',psi,'resolution',1*degree,...
  'interface','generic',...
  'ColumnNames', { 'phi1' 'Phi' 'phi2' 'Weight'}, 'Columns',[1 2 3 4], 'Bunge');

ori_s = orientation.byMiller([1 2 3],[6 3 4],CS,SS);
volume_s =  volume(odf,ori_s,10*degree)       % Here the ans is 0.07


—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1765>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN4KKFVZVAAN565HXX3W34LWRANCNFSM6AAAAAAVZBSAYY>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1765", "url": "https://github.com/mtex-toolbox/mtex/discussions/1765", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by Brian-Liu-rx** [reply]
Hi,   Luiz , 
    Do you nean that the original data i input is using a different SS with that I give to S component?
    plus, for both case I give the same SS to  odf and S component.

    Thanks!

**Reply by Brian-Liu-rx** [reply]
Which one is correct for the rolled Al？

**Comment by lfgmorales** [comment]
Hi

I don’t know about your data, but depending on the way your sample was “deformed”, it might have a triclinic or orthorhombic symmetry (the later very typical in metals for example)

Have a look here

<https://mtex-toolbox.github.io/DetectionOfSampleSymmetry.html>
Detection of Sample Symmetry | MTEX<https://mtex-toolbox.github.io/DetectionOfSampleSymmetry.html>
mtex-toolbox.github.io<https://mtex-toolbox.github.io/DetectionOfSampleSymmetry.html>
        [favicon.png] <https://mtex-toolbox.github.io/DetectionOfSampleSymmetry.html>

Cheers

L

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 13 Mar 2023, at 14:26, Brian-Liu-rx ***@***.***> wrote:

﻿

Hi, Luiz ,
Do you nean that the original data i input is using a different SS with that I give to S component?
plus, for both case I give the same SS to odf and S component.

Thanks!


—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1765#discussioncomment-5291578>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN2ULZRVO7AASYXREM3W34N6HANCNFSM6AAAAAAVZBSAYY>.
You are receiving this because you commented.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1765#discussioncomment-5291578", "url": "https://github.com/mtex-toolbox/mtex/discussions/1765#discussioncomment-5291578", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by Brian-Liu-rx** [reply]
Thank you Luiz. Orthorhombic seems good.
